packages/generate-newsletter/index.js

```diff
@@ -0,0 +1,98 @@
+const fs = require('fs-extra')
+const path = require('path')
+const htmlCreator = require('html-creator')
+const { prompt } = require('enquirer')
+const { extractSheets } = require('spreadsheet-to-json')
+const argv = require('yargs').argv
+
+const { getRandomColor, colorLuminance, getCurrentDate } = require('./utils')
+const credentials = require('./credentials.json')
```

Comment:
can we add a empty file?

Reply:
yeah.. but the file is being added to gitignore.. so how we should do it?

Reply:
Added a different condition to check this one.

Reply:
Aww. ya lets leave it then. We can update in readme. Things needed to generate newsletter